examples/frameworks/strands_demo/bedrock_agentcore/README.md

```
@@ -0,0 +1,860 @@
+4. Click **Create role**
+
+### Step 7: Record the Role ARN
+
+After the role is created, you'll be redirected to the Roles page:
+
+1. In the search box, type: `AgentCore_NAT`
+2. Click on the **AgentCore_NAT** role name
+3. On the role summary page, locate and copy the **ARN** (Amazon Resource Name)
+
+The ARN will look like this:
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)